File: 백준/Silver/4375. 1/1.py
import sys
input = sys.stdin.readline

while True:
    try:
        n = int(input().strip())
    except:
        break
    remainder = 1  
    cnt = 1  
    while remainder % n != 0:
        remainder = (remainder * 10 + 1) % n 
        cnt += 1

    print(cnt)

File: 백준/Silver/4375. 1/1.java
import java.util.*;
import java.io.*;

public class Main{
    public static void main(String[] args) throws IOException{
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        String input;

        while((input =br.readLine())!=null){
            int n = Integer.parseInt(input.trim());
            int remainder = 1;
            int cnt =1;
            while(remainder %n!=0){
                remainder = (remainder *10+1)%n;
                cnt ++;
            }
            System.out.println(cnt);
        }
    }
}